tests/test_smoke.py

```diff
@@ -668,7 +691,30 @@ def test_cli_logs(generic_cloud: str):
     run_one_test(test)
 
 
+@pytest.mark.lambda_labs
+def test_lambda_logs():
```

Comment:
nit: Add a `TODO: merge this with 'test_cli_logs' when multi-node is supported.`

Reply:
Added!